casdoorsdk/enforce.go

```diff
@@ -44,6 +43,23 @@ func Enforce(permissionRule *PermissionRule) (bool, error) {
 		return false, err
 	}
 
+	if len(bytes) == 0 {
```

Comment:
Plz merge the duplicated code of these two functions into `doEnforce()`

Reply:
plz review